### DISCUSSION TITLE
Copilot for PRs Failing GitHub Checks

### DISCUSSION THREAD

**Discussion by sayhiben** [community/community#60281] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

This afternoon, my team began to experience issues with Copilot for PRs wherein the Copilot checks fail:

>## Analysis complete
>### Summary
>No summary available (An error occurred while summarizing these changes: Gave up after 3 retries: Failed to read error response)
> 
>### Walkthrough
>No walkthrough available (An error occurred while summarizing these changes: Gave up after 3 retries: Failed to read error response)

```
"Error: Gave up after 3 retries: Failed to read error response\n at Object.getOneUnstreamedCompletion (/home/site/wwwroot/lib/src/engine/completions.js:112:15)\n at async Object.overallWalkthrough (/home/site/wwwroot/lib/src/engine/describe.js:665:35)\n at async Promise.all (index 0)\n at async generateDescriptionsWorker (/home/site/wwwroot/lib/src/bot/bot-describe.js:336:36)\n at async DataStore.cache (/home/site/wwwroot/lib/src/engine/datastore.js:178:34)"
"Error: Gave up after 3 retries: Failed to read error response\n at Object.getOneUnstreamedCompletion (/home/site/wwwroot/lib/src/engine/completions.js:112:15)\n at async Object.overallSummary (/home/site/wwwroot/lib/src/engine/describe.js:571:31)\n at async Promise.all (index 1)\n at async generateDescriptionsWorker (/home/site/wwwroot/lib/src/bot/bot-describe.js:336:36)\n at async DataStore.cache (/home/site/wwwroot/lib/src/engine/datastore.js:178:34)"
```

**Comment by Akash1134** [comment]
Hi @sayhiben ,

Thanks for reporting the issue. I have shared this with my team internally.

**Comment by Akash1134** [comment, marked as answer]
Hi @sayhiben ,

I checked with my team and this sounds like a temporary one time glitch. Could you please  retry (edit the description and put the tags back in) and let us know if the issue still persists? 

Thanks!
